sormas-api/src/main/java/de/symeda/sormas/api/hospitalization/HospitalizationDto.java

```diff
@@ -17,16 +17,16 @@
  *******************************************************************************/
 package de.symeda.sormas.api.hospitalization;
 
-import java.util.ArrayList;
```

Comment:
Please setup automatic code formatting as described here: https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/DEVELOPMENT_ENVIRONMENT.md. There should not be any changes like this in the files.